pkg/controlplane/rest/accesspolicy.go

```diff
@@ -29,6 +32,17 @@ func accessPolicyToAPI(policy *store.AccessPolicy) *api.Policy {
 	return &policy.Policy
 }
 
+// accessPolicyFromBlob unmarshals an AccessPolicy object encoded as json in a byte array.
```

Comment:
I understand that defining Privileged in terms of normal access policy can be useful (e.g., less code), but it also creates additional code (e.g., marshalling blobs instead of via apimachinery directly). It's somewhere in the area of "it's almost the same thing, but only 90% so..." so I wonder if you want to keep them more separate or not in the code (structures and logic paths). I don't have a strong opinion, but think it could be somewhat confusing for someone who sees the code for the first time (e.g., the note about the two structs have to be the same or things break...)

Reply:
REST API is a throw-away code, so probably no need to make it extra pretty.